app/components/concept-page/content.ts

```diff
@@ -33,11 +41,23 @@ export default class ContentComponent extends Component<Signature> {
     this.currentProgressPercentage = 0;
   }
 
+  @action
+  handleCopyButtonClick() {
+    this.analyticsEventTracker.track('clicked_on_share_concept_button', {
+      concept_id: this.args.concept.id,
+    });
+  }
+
   @action
   handleProgressPercentageChanged(progressPercentage: number) {
     this.currentProgressPercentage = progressPercentage;
   }
 
+  @action
+  handleShareConceptContainerInserted(element: HTMLElement) {
+    element.scrollIntoView({ behavior: 'smooth', block: 'nearest' });
```

Comment:
Looks like we'll be inserting three separate elements and calling `scrollIntoView` on them - completion container, next concept and share concept.. Can we just scroll one instead? Maybe we could place both "next concept" and "share concept" within "completion container", and only keep the scroll handler that's present there? That way the whole block will be scrolled into view.